ui/app/components/app/transaction-list-item-details/transaction-list-item-details.component.js

```diff
@@ -33,6 +33,8 @@ export default class TransactionListItemDetails extends PureComponent {
     cancelDisabled: false,
   }
 
+  handleRetryClick = event => this.handleRetry(event)
```

Comment:
Why did you create `handleRetryClick`? Any reason to not just use `this.handleRetry` on line 181?

Reply:
no reason, really. I originally had a separate method and then just chained them together... more placeholder than anything else. had a feeling this was going to need more work